packages/salesforcedx-vscode-core/src/commands/util/parameterGatherers.ts

```diff
@@ -340,3 +340,28 @@ export class MetadataTypeGatherer extends SimpleGatherer<{ type: string }> {
     super({ type: metadataType });
   }
 }
+
+export class PromptConfirmGatherer
+  implements ParametersGatherer<{ choice: string }> {
+  private question: string;
+
+  constructor(question: string) {
+    this.question = question;
+  }
+  public async gather(): Promise<
+    CancelResponse | ContinueResponse<{ choice: string }>
+  > {
+    const confirmOpt = nls.localize('parameter_gatherer_prompt_confirm_option');
+    const cancelOpt = nls.localize('parameter_gatherer_prompt_cancel_option');
+    const choice = await this.showMenu([confirmOpt, cancelOpt]);
```

Comment:
Nit, but should cancel be the first/default option since this is a destructive action?

Reply:
That's a good point, let me make the change 👍